docs/FAQ.md

```diff
@@ -1,11 +1,24 @@
-# FAQ
+# Frequently Asked Questions
 
-### What is Project Beman?
+## What is the relationship between the Beman Project and the Boost Foundation?
 
+While both Beman Project and Boost strive to provide high-quality C++ libraries to the community, the Beman Project is specifically focused on fostering works that can be adopted by the standard library.
 
-### Why would I want to participate in Project Beman?
+## Do I have to write a WG21 C++ Standards Committee Paper before I can contribute to the Beman Project?
 
+No! The Beman Project wants to work with you, with experience, feedback, and expertise. Obviously, a paper must be written before becoming accepted as part of the standard, but that paper can be written in tandem with your library development.
 
-### What is the relationship between Project Beman and Boost Foundation?
+In fact, there are plenty of papers already written that need an implementation before they can progress in through the standards track. You can contribute to the Beman project by finding a paper that interests you and working on its library.
 
+Please reach out to the Beman Project Leads to help get your specific project started.
 
+## I have a crazy library that’s probably never going to be a standard library feature, but I use it and think it’s neat. Can I contribute it to the Beman Project?
+
+We’re glad you’ve developed a neat and useful library, but the Beman Project is dedicated to getting the highest quality libraries through the rigorous effort of standardization.
```

Comment:
I think rewording this Q&A to remove words like "neat" and "crazy" would be appropriate. I could see a reader taking this as not being worded in good faith.

It might also be worth pointing the reader at the Discourse. Hashing out library viability in an interactive context where people can get to know one another seems helpful.

Reply:
@bretbrownjr I attempted to address this concern in https://github.com/beman-project/beman/pull/20. LMK what you think there.